tests/integration/side-navigation.test.js

```diff
@@ -0,0 +1,40 @@
+import {waitFor, screen} from '@testing-library/dom';
+import userEvent from '@testing-library/user-event';
+import expandableDocsHTML from '../../templates/docs/examples/patterns/side-navigation/_expandable-docs.html';
+
+function render(html, scriptUrl = '../../templates/docs/examples/patterns/side-navigation/_expand_script') {
+  const container = document.createElement('div');
+  container.innerHTML = html;
+  document.body.appendChild(container);
+  import(scriptUrl);
+  return {container};
+}
+
+it('adds aria-expanded to list elements on load', async () => {
+  const {container} = render(`
+  <ul class="p-side-navigation__list">
```

Comment:
Does this part of HTML need to be hardcoded? Or can it be included from some example?

Reply:
Could be imported if you think that works better, added this as an alternative - it's quite nice to see it all in a single place.